docs: bulk delete alert rule group (#105087)
grafana/grafana: docs/sources/alerting/alerting-rules/create-grafana-managed-rule.md

{{< docs/shared lookup="alerts/configure-notification-message.md" source="grafana" version="<GRAFANA_VERSION>" >}}

> ## Bulk delete all alert rules for a rule group

Reviewer: ```suggestion
## Bulk delete all alert rules within a folder
```

> Admin users can delete all of the alert rules for a rule group. To delete all the alert rules for a rule group folder, click the menu icon and select **Delete**. Then type "Delete" into the field and click **Delete** to confirm the bulk deletion. 

Reviewer: ```suggestion
Admin users can delete all of the alert rules within a folder. To delete all the alert rules in a folder, click the menu icon and select **Delete**. Then type "Delete" into the field and click **Delete** to confirm the bulk deletion. 
```
